README.md

```diff
@@ -4,12 +4,13 @@ xSTUDIO is a media playback and review application designed for professionals wo
 
 ## Building xSTUDIO
 
-This release of xSTUDIO can be built on various Linux flavours. MacOS and Windows compatibility is not available yet but this work is on the roadmap for 2023.
+This release of xSTUDIO can be built on various Linux flavours. MacOS compatibility is not available yet but this work is on the roadmap for 2023.
 
-We provide comprehensive build steps for 3 of the most popular Linux distributions.
+We provide comprehensive build steps for 4 of the most popular Linux distributions.
 
 * [CentOS 7](docs/build_guides/centos_7.md)
 * [Rocky Linux 9.1](docs/build_guides/rocky_linux_9_1.md)
 * [Ubuntu 22.04](docs/build_guides/ubuntu_22_04.md)
+* [Windows](docs/build_guides/windows.md)
```

Comment:
Windows has been mentioned under Linux distributions.
This can be rephrased to accommodate Linux and Windows both.
Alternatively, both may have their own separate sections.

For example (Markdown):

```
## Building xSTUDIO

This release of xSTUDIO can be built on various Linux flavours and Windows too.
MacOS compatibility is not available yet but this work is on the roadmap for 2023.

### Linux

* [CentOS 7](docs/build_guides/centos_7.md)
* [Rocky Linux 9.1](docs/build_guides/rocky_linux_9_1.md)
* [Ubuntu 22.04](docs/build_guides/ubuntu_22_04.md)

### Windows

* [Windows](docs/build_guides/windows.md)
```

---

Preview:

## Building xSTUDIO

This release of xSTUDIO can be built on various Linux flavours and Windows too.
MacOS compatibility is not available yet but this work is on the roadmap for 2023.

### Linux

* [CentOS 7](docs/build_guides/centos_7.md)
* [Rocky Linux 9.1](docs/build_guides/rocky_linux_9_1.md)
* [Ubuntu 22.04](docs/build_guides/ubuntu_22_04.md)

### Windows

* [Windows](docs/build_guides/windows.md)

---

Reply:
Apparently, https://github.com/AcademySoftwareFoundation/xstudio?tab=readme-ov-file#welcome-to-xstudio already have separate sections for Linux, Windows and MacOS.
So, this README may need to be updated accordingly. Thanks!

Reply:
Thanks, I've updated both the build doc and readme to be better organized and more up-to-date with the current state.